app/controllers/new_admin/questionnaires_controller.rb

```diff
@@ -67,6 +68,16 @@ def questionnaire_params
                                             questions_attributes: %i[title kind raw_answer_options _destroy id])
     end
 
+    def questions_to_be_destroyed_ids
+      return nil unless questionnaire_params[:questions_attributes]
+      ids = questionnaire_params[:questions_attributes].values.filter_map do |question|
+        if question["_destroy"] != 'false'
+          question["id"]
+        end
+      end
+      ids
```

Comment:
Using a temp var here is unnecessary. `filter_map` will already return a list of filtered ids:
```suggestion
      questionnaire_params[:questions_attributes].values.filter_map do |question|
        if question["_destroy"] != 'false'
          question["id"]
        end
      end
```

Reply:
done